force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -269,15 +430,31 @@ public with sharing class FlowControlLoopsCollections {
      * @param words The list of words to search.
      * @return The position of the word in the list as a string or -1 if the word was not found.
      */
-    public static Integer findWordInList(String wordToFind, List<String> words) {
+
+      public static Integer findWordInList(String wordToFind, List<String> words) {
+
         // The variable to store the index of the word
 
+        Integer wordIndex = -1 ;  //default is -1, or NOT found
+
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
 
-        return null; // Replace null with the variable you used to store the result
+        for (String word : words) {
+            if (word.equals(wordToFind)) {
+                wordIndex = words.indexOf(word) ;
```

Comment:
If you are going to use indexOf() you dont need to loop through the words at all. If searches the whole list anyways. Right now you could just be searching multiple times.